docs/application/web/guides/data/file-system.md

```diff
@@ -108,149 +106,256 @@ To manage file storages:
 
-You can retrieve file content as a `DOMString` with the `readAsText()` method. The encoding input parameter of the method defines the format in which the file content is returned.
+1. To retrieve a list of all files and directories located in a specified directory, use the `listDirectory()` method of the `filesystem` object. The method returns an array of `DOMString` elements, each beeing a path with name of file inside a directory pointed by `path`.
 
-To get files and file details from the file system:
+   ```
+   function onsuccess(files) {
+       for (var i = 0; i < files.length; i++) {
+           // Display the file path with name
+           console.log('File path and name is: ' + files[i]);
+       }
+   }
+   function onerror(error) {
+       console.log(error);
+   }
+   tizen.filesystem.listDirectory('documents/subDir', onsuccess, onerror);
+   ```
 
-1. To access a specific file or directory within the file system, retrieve a file handle using the `resolve()` method of the `FileSystemManager` interface (in [mobile](../../../../org.tizen.web.apireference/html/device_api/mobile/tizen/filesystem.html#FileSystemManager), [wearable](../../../../org.tizen.web.apireference/html/device_api/wearable/tizen/filesystem.html#FileSystemManager), and [TV](../../../../org.tizen.web.apireference/html/device_api/tv/tizen/filesystem.html#FileSystemManager) applications):
+2. To retrieve the file URI, use the `toURI()` method:
 
    ```
-   tizen.filesystem.resolve('documents', onResolveSuccess, null, 'r');
+   function onsuccess(files) {
+       for (var i = 0; i < files.length; i++) {
+           // Display the file URI
+           console.log('File URI is: ' + toURI(files[i]));
+       }
+   }
+   function onerror(error) {
+       console.log(error);
+   }
+   tizen.filesystem.listDirectory('documents/subDir', onsuccess, onerror);
    ```
 
-   The `File` object (in [mobile](../../../../org.tizen.web.apireference/html/device_api/mobile/tizen/filesystem.html#File), [wearable](../../../../org.tizen.web.apireference/html/device_api/wearable/tizen/filesystem.html#File), and [TV](../../../../org.tizen.web.apireference/html/device_api/tv/tizen/filesystem.html#File) applications) is returned in the success event handler.
+3. To retrieve the file content as a DOMString, binary data or to put it's contents to a `Blob` object use one of the following functions on `FileHandle` object opened in modes appropriate for reading ('r', 'rw').
 
-2. To retrieve a list of all the files and their directories located in a specified directory, use the `listFiles()` method of the `File` object:`function onResolveSuccess(dir) {    dir.listFiles(onsuccess);}`The method returns an array of `File` objects.
-
-3. To retrieve the file URI, use the `toURI()` method:
+   - First open a file using `openFile()` function with mode allowing to read it's contents.
+   ```
```

Comment:
Would you please add 3 more spaces in front of the code, from line 194 to line 199?

This sample code box is sub item of `First open a file using openFile()~` I think, so the layout should be fixed a little.

Reply:
Done.